WcaOnRails/config/routes.rb

```diff
@@ -98,6 +98,11 @@
     get '/admin/results/:round_id/new' => 'admin/results#new', as: :new_result
   end
 
+  get 'payment/config' => 'payment#payment_config'
+  get 'payment/finish' => 'payment#payment_finish'
+  get 'payment/refunds' => 'payment#available_refunds'
+  get 'payment/refund' => 'payment#payment_refund'
```

Comment:
Can you scope this? cf. https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing